rfcs/0002-upload-from-dropbox/text.md

```diff
@@ -164,11 +165,13 @@ As the data is downloaded, the hash of the file will be calculated as it arrives
 [checksumming_io](https://github.com/HumanCellAtlas/checksumming_io) can be used. If the final hash does not match,
 then the download process will be retried.
 
-While downloading the file, the container will periodically check if the job has been canceled using the upload
-table. If the upload status is _Cancel Pending_, the upload container will stop the stream, delete any data that has been downloaded,
+Roughly every 10 seconds, the container will check the DP database to see if the `upload_status` been changed to "Cancel
```

Comment:
I think we need something in there. There were multiple questions in the meeting about what "periodically" meant, which I think is sensible because it could mean "several times per second" to "once or twice per job". So including some order of magnitude information is helpful. Maybe I can make the "roughly" a bit stronger?